Kinder/Swiping.xaml.cs

```diff
@@ -30,28 +30,31 @@ public Swiping()
         {
             InitializeComponent();
             LoadItems();
+            RenderItem();
         }
 
         private void LoadItems()
         {
             //loading all items:
-            StreamReader file = new(FileLocation_items);
+            StreamReader file1 = new(FileLocation_items);
```

Comment:
Maybe we will do path names start with uppercase letter? Like file1 should be File1 to match the single standart :)